release/models/system/openconfig-system-management.yang

```diff
@@ -127,6 +127,98 @@ module openconfig-system-management {
     }
   }
 
+
+  grouping system-netconf-server-config {
+    description
+      "Configuration data for the netconf server/subsystem";
+    leaf enable {
+      type boolean;
+      default false;
+      description
+        "Enables the netconf server. The gRPC server is enabled by
```

Comment:
I think you can just say that this leaf enables the NETCONF server -- I don't think we need to indicate that gRPC is enabled here.

Reply:
@aashaikh Sorry for  replying late . have updated the comment given by you .
Please review and help in further steps